CODEOWNERS

```diff
@@ -1,2 +1 @@
-*       @coemans
-*       @pascalriquier
\ No newline at end of file
+* @coemans @pascalriquier
```

Comment:
dit is verkeerd, ik moet altijd eerst staan :-)

Reply:
Je staat eerst <sub><sup>als je vanachter begint te tellen.</sub></sup>